src/components/Items/ListItem.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <q-item v-ripple clickable>
+    <q-item-section @click="loadList()">
+      <q-input
+        ref="listName"
+        v-model="currentList.name"
+        @blur="isEdit = false"
+        type="text"
+        :readonly="!isEdit"
+      />
+    </q-item-section>
+    <q-item-section side>
+      <q-item-label caption
+        >{{ currentList.cartTotal || 0 }}/{{
```

Comment:
TODO: The total price and the total number of items doesn't change when the list is updated